### PR TITLE
Added a field for a students name.

### PR DESCRIPTION
**Pull request by galileokim**
Please don't be mean.

### CODE REVIEW

File: src/people/Student.java [hunk 1]
```diff
@@ -3,5 +3,14 @@
 public class Student {
     // name
     // grade
+    private final String name;
```
Comment (JerryBao98) [inline src/people/Student.java]: Please remove the final, and use a setter.

Comment (JerryBao98) [reply src/people/Student.java]: `    public void setName(String name){
        this.name = name;
    }`

**Review by JerryBao98 — APPROVED** [review]
Looks good. Nice job.

**Review by destroyer8898 — APPROVED** [review]
blessed code

**Review by danielquzhao — APPROVED** [review]
very cool

**Pull request merged**